proposed/2022/DotnetListPackageMachineReadableJsonOutput.md

```diff
@@ -772,6 +772,139 @@ Outputs json for format version 1, if it's not specified then latest version'll
 }
 ```
 
+### Error handling
+
+In case of an error it would be written into `problems` section. But in case there is parameter, options error then it may defaults back to console output instead of json output.
```

Comment:
`npm -ls json` does same thing it creates `problems section` at top of json output. Only problem with this approach is json output needs to be completed internally before it displays on console.